DEVELOPING.md

```diff
@@ -0,0 +1,76 @@
+# Developing Beluga
```

Comment:
Note that some repository-level documentation has been kept around. What looks good in Github doesn't look good in Sphinx and viceversa. The idea here is to keep these READMEs around as an addendum to code for direct contributors.